site/index.md

```diff
@@ -35,6 +35,11 @@ images:
 {:.hidden}
 # Vega-Lite: a high-level visualization grammar
```

Comment:
This hidden link was there for seo, I think. Can you use it instead of the new one below. 

```
{:.page-centered}
```

applies a class as you can see.